bgpd/bgp_bmp.c

```diff
@@ -120,16 +120,12 @@ DECLARE_LIST(bmp_mirrorq, struct bmp_mirrorq, bmi);
 static int bmp_listener_cmp(const struct bmp_listener *a,
 		const struct bmp_listener *b)
 {
-	int c;
-
-	c = sockunion_cmp(&a->addr, &b->addr);
-	if (c)
-		return c;
 	if (a->port < b->port)
 		return -1;
 	if (a->port > b->port)
 		return 1;
-	return 0;
+
```

Comment:
Why the order change? What's the goal here?

Reply:
I was just looking for the core and sticked on how peer_cmp() or nhgc_addr_cmp_helper() does. Basically NOOP here, I can drop it.

Reply:
Dropped.